### PR TITLE
[Vercel] [NS Records] Update gunman.json and _vercel.gunman.json

### PR DESCRIPTION
**Pull request by MukuJhansi**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

I need to use NS Records to Change Records Quickly as I am constantly switching between services and thinking to make a nested subdomain to test all sort of thing (i.e. Mail Server ,A FTP Service)but with manual pull Update it wastes my time as some service require a domain to Even Test Their Service(s). I am a Honest User and I assure you that I will bear full responsibility for any sort of mis-happenings.

I Would Be Thankful for you to even considering my Request. If Issuing NS Records is Not Possible then I kindly request you to Merge My Service Change Request [_vercel.gunman.json]
Thanks

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://main-mukujhansis-projects.vercel.app
image

### CODE REVIEW

File: domains/_vercel.gunman.json
Comment (DEV-DIBSTER) [inline domains/_vercel.gunman.json:1]: If you get NS records, then you wouldn't need this file FYI,.

Comment (DEV-DIBSTER) [inline domains/_vercel.gunman.json:1]: If you get NS records, you wouldn't need this file.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by wdhdev** [comment]
i don't think NS records will be accepted for this

**Comment by MukuJhansi** [comment]
> i don't think NS records will be accepted for this

That is Certainly fine just please approve the _vercel file so that domain would be accessible. [and delete any records in gunman.json]

File: domains/_vercel.gunman.json
Comment (MukuJhansi) [reply domains/_vercel.gunman.json:1]: I added it in case if NS records didn't got approved then I don't need to create a new Pull Request

**Review by wdhdev — APPROVED** [review]

**Pull request merged**